examples/iot/v1beta1/domainconfiguration.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: iot.aws.upbound.io/v1beta1
+kind: DomainConfiguration
+metadata:
+  annotations:
+    meta.upbound.io/example-id: iot/v1beta1/domainconfiguration
+  labels:
+    testing.upbound.io/example-name: iot
+  name: iot
+spec:
+  forProvider:
+    domainName: iot.example.com
+    region: us-west-1
+    serverCertificateArnsSelector:
+      matchLabels:
+        testing.upbound.io/example-name: certificate
+    serviceType: DATA
+---
+apiVersion: acm.aws.upbound.io/v1beta2
+kind: Certificate
+metadata:
+  annotations:
+    meta.upbound.io/example-id: acm/v1beta2/certificate
```

Comment:
This should be set to the file name of the example. It's a means of identifying which is the "primary" resource in an example manifest

Reply:
good to know, thanks! I changed it in both files